### PR TITLE
[CP] 3.18 cherry-picks

### PR DESCRIPTION
**Pull request by jturel**
the 3.18 branch was cut from a commit a bit too early - this brings over what's missing and will likely fix the broken packer build

### CODE REVIEW

File: manifests/params.pp [hunk 1]
```diff
@@ -41,6 +41,9 @@
   $fork_remote_name = undef
   $upstream_remote_name = 'upstream'
 
+  $foreman_scm_revision = 'develop'
+  $katello_scm_revision = 'master'
```
Comment (ekohl) [inline manifests/params.pp:45]: Shouldn't these be 3.18 and whatever Foreman matches that? (2.3?) That way the defaults are correct if you check out the branch.

Comment (jeremylenz) [inline manifests/params.pp:45]: I wonder if there's a list of places we need to change when this changes to `main`..

Comment (jturel) [reply manifests/params.pp:45]: They get overridden by our packer scripting via installer flags, I don't think anyone is directly checking these branches out ... yet

**Review by jeremylenz — APPROVED** [review]
LGTM

File: manifests/params.pp (hunk 1 above)
Comment (jturel) [reply manifests/params.pp:45]: Nonexistent afaik

**Comment by jturel** [comment]
@ekohl any idea what's up with the unit tests here?

File: metadata.json [hunk 2]
```diff
@@ -30,7 +30,7 @@
     },
     {
       "name": "katello-katello",
-      "version_requirement": ">= 6.1.0"
+      "version_requirement": "<= 16.0.0"
```
Comment (jturel) [inline metadata.json:33]: @ekohl does this pinning not affect the dependencies used by the unit tests here?

Comment (ekohl) [reply metadata.json:33]: No, that's in `.fixtures.yml`.

**Comment by jturel** [comment]
Getting somewhere. need to pin more things

**Pull request closed without merging**